lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -324,9 +324,6 @@ def set_aspect(self, aspect, adjustable=None, anchor=None, share=False):
         """
         _api.check_in_list(('auto', 'equal', 'equalxy', 'equalyz', 'equalxz'),
                            aspect=aspect)
-        if adjustable is None:
-            adjustable = self._adjustable
```

Comment:
I think we still need to keep these lines in to set a value for `adjustable` if it's passed as `None`?